console/console-init/ui/src/modules/address-detail/components/AddressDetailHeader/AddressDetailHeader.tsx

```diff
@@ -61,6 +63,8 @@ const styles = StyleSheet.create({
 export const AddressDetailHeader: React.FunctionComponent<IAddressDetailHeaderProps> = ({
   type,
   topic,
+  Deadletters,
```

Comment:
please revisit this name. All props will be camel cased. like `deadLetter` and `expiryQueue`